src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -73,7 +81,7 @@ public async Task<bool> ProcessPayload(WorkflowRequestEvent message)
 
             var workflowInstances = new List<WorkflowInstance>();
 
-            workflows.ForEach((workflow) => workflowInstances.Add(CreateWorkFlowIntsance(message, workflow)));
+            workflows.ForEach(async (workflow) => workflowInstances.Add(await CreateWorkflowInstance(message, workflow)));
```

Comment:
CreateWorkflowInstance
Should maybe be called CreateWorkflowInstanceAsync now?

Reply:
Good shout, have changed it to that.